latency/src/bin/z_ping.rs

```diff
@@ -37,125 +39,178 @@ struct Opt {
     interval: f64,
     #[structopt(long = "parallel")]
     parallel: bool,
+    #[structopt(short = "d", long = "samples")]
+    samples: Option<usize>,
 }
 
-async fn parallel(opt: Opt, config: Properties) {
-    let zenoh = Zenoh::new(config.into()).await.unwrap();
-    let zenoh = Arc::new(zenoh);
+async fn single(opt: Opt, config: Config) {
```

Comment:
The order of the function is same with the file `zn_ping` (I modified the `zn_ping` to the newer `z_ping`, and remove the older `z_ping`), so I think may be we don't  need to change the order of the function anymore.

Reply:
The comparison will be based on z_*. So, making them aligned together on z* instead of aligning with zn* will be better. Also, that way will keep credits for the original developers. Thanks.

Reply:
Ok, I had recovered the `zn*` files and edited `z*` files.